docs/release-notes/8.7.asciidoc

```diff
@@ -1,6 +1,22 @@
 [[release-notes-header-8.7.0]]
 == 8.7
 
+[discrete]
+[[release-notes-8.7.1]]
+=== 8.7.1
+
+[discrete]
+[[enhancements-8.7.1]]
+==== Enhancements
+There are no user-facing changes in 8.7.1.
+
+[discrete]
+[[bug-fixes-8.7.1]]
+==== Bug fixes
+* Fixes a bug that caused the {agent} upgrade from 8.6.2 to 8.7.0 to fail for {agents} running the Osquery Manager integration (https://github.com/elastic/elastic-agent/pull/2448[#2448]).
```

Comment:
we should note here that upgrade `8.7.0` -> `8.7.1` could still cause an upgrade issue.  We should include the same workarounds as below.  After the upgrade to `8.7.1`, the problem will be fixed.

I'm referring to this section from 8.7.0 known issues.

```
The {agent} upgrade from 8.6.2 to 8.7.0 might fail for {agents} running the Osquery Manager integration (https://github.com/elastic/elastic-agent/issues/2433[#2433]). To prevent this, delete the Osquery Manager integration from your {agents} before upgrading them to 8.7.0, then re-add it after upgrading
```